src/components/menu/MenuList.tsx

```diff
@@ -125,22 +139,27 @@ const MenuListComponent = () => {
     [menuProps]
   );
 
+  if (!isVisible)
+    return null
+
   return (
-    <AnimatedView
-      intensity={100}
-      animatedProps={animatedProps}
-      style={[styles.menuContainer, messageStyles]}
-    >
-      <Animated.View
-        style={[
-          StyleSheet.absoluteFillObject,
-          styles.menuInnerContainer,
-          animatedInnerContainerStyle,
-        ]}
+    <Animated.View style={[styles.menuContainer, messageStyles]}>
+      <AnimatedView
+        intensity={100}
+        animatedProps={animatedProps}
+        style={StyleSheet.absoluteFillObject}
       >
-        <MenuItems items={itemList} />
-      </Animated.View>
-    </AnimatedView>
+        <Animated.View
+          style={[
+            StyleSheet.absoluteFillObject,
+            styles.menuInnerContainer,
+            animatedInnerContainerStyle,
+          ]}
+        >
```

Comment:
what does this change do?

Reply:
This is applying the fix from this PR: https://github.com/enesozturk/react-native-hold-menu/pull/112/files